### PR TITLE
Enable rebuild triggers

### PR DESCRIPTION
**Pull request by bfontecc007**
Enable autorebuild by adding ImageChange trigger and activating import_image plugin.

### CODE REVIEW

File: inputs/orchestrator_inner:5.json [hunk 1]
```diff
@@ -129,6 +129,16 @@
...
+    },
+    {
+      "args": {
+        "imagestream": "{{IMAGESTREAM}}",
+        "docker_image_repo": "{{DOCKER_IMAGE_REPO}}",
+        "url": "{{OPENSHIFT_URI}}",
+        "verify_ssl": false,
+        "build_json_dir": "{{PATH}}"
+      },
+      "name": "import_image"
```
Comment (lcarva) [inline inputs/orchestrator_inner:5.json]: Let's list the name first like the other plugins.

File: tests/build_/test_arrangements.py [hunk 2]
```diff
@@ -1198,6 +1198,7 @@ class TestArrangementV5(TestArrangementV4):
                 'group_manifests',
                 'pulp_tag',
                 'pulp_sync',
+                'import_image',
```
Comment (lcarva) [inline tests/build_/test_arrangements.py]: Do we need a test to ensure the plugin parameters were rendered correctly?

Comment (bfontecc007) [reply tests/build_/test_arrangements.py]: I'll code it if you think we need it.

**Comment by lcarva** [comment]
Updated release notes draft:
> - ImageChangeTriggers enabled
> - `import_image` plugin enabled

**Pull request merged**